utils/default-helpers/src/main/scala/pl/touk/nussknacker/engine/util/functions/conversion.scala

```diff
@@ -20,4 +22,116 @@ trait ConversionUtils extends HideToString {
   def toNumber(@ParamName("stringOrNumber") stringOrNumber: Any): java.lang.Number =
     numeric.toNumber(stringOrNumber)
 
+  @Documentation(description = "Parse any value to Number or null in case failure")
+  def toNumberOrNull(@ParamName("value") value: Any): java.lang.Number = value match {
+    case v: String => Try(numeric.toNumber(v)).getOrElse(null)
+    case v: Number => v
+    case _         => null
+  }
+
+  @Documentation(description = "Convert any value to String")
+  def toString(@ParamName("value") value: Any): java.lang.String = value match {
+    case v: String => v
+    case null      => null
+    case v         => v.toString
+  }
+
+  @Documentation(description = "Convert any value to Boolean or throw exception in case of failure")
+  def toBoolean(@ParamName("value") value: Any): java.lang.Boolean = value match {
+    case v: String            => java.lang.Boolean.valueOf(v)
```

Comment:
IMO `java.lang.Boolean.valueOf` method is tricky. Because `java.lang.Boolean.valueOf("nu") == false` ;) I'm not sure if this is what we want here

Reply:
Done